config/initializers/feature_toggle.rb

```diff
@@ -20,4 +20,10 @@ def self.location_toggle_enabled?
 
     false
   end
+
+  def self.new_locations_table_layout_enabled?
+    return true unless Rails.env.production?
+
+    false
+  end
```

Comment:
I know this is just duplicating previous methods (I did the same for a scheme_toggle one) but these could probably be 
```suggestion
def self.location_toggle_enabled?
    !Rails.env.production?
  end
```
etc.

Reply:
(I've changed the other ones here but if you could update yours too that would be good I think https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/pull/980/files#diff-a32908531df21028fc3e01feaacaf51a0a6f36f976d0a1e13676e8d1d5caf20aR6)